test/plugins/individual-collateral/morpho-aave/MorphoAaveV2TokenisedDeposit.test.ts

```diff
@@ -212,11 +252,57 @@ const execTestForToken = ({ token, poolToken, amount }: ITestSuiteVariant) => {
       await closeTo(methods.assets(bob), fraction(50))
     })
 
+    it('Check that inflation attacks are not possible', async () => {
```

Comment:
Let's use the same pattern from here when it comes to naming / documentation: https://github.com/reserve-protocol/protocol/blob/11aa80b8235689484c84b452c7e1145fcef988d7/test/Furnace.test.ts#L450